shared/src/store/action-reducers/transfer-point.ts

```diff
@@ -256,12 +256,14 @@ export namespace TransferPointActionReducers {
                     draftState.transferPoints
                 )) {
                     const transferPoint =
-                        draftState.transferPoints[_transferPointId];
+                        draftState.transferPoints[_transferPointId]!;
                     for (const connectedTransferPointId of Object.keys(
                         transferPoint.reachableTransferPoints
                     )) {
                         const connectedTransferPoint =
-                            draftState.transferPoints[connectedTransferPointId];
+                            draftState.transferPoints[
```

Comment:
Use `Object.entries()` instead

Reply:
I don't know what you mean here.

Reply:
Never mind, `reachableTransferPoints` are only the ids...